app/signals/apps/classification/models/classifier.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.gis.db import models
+
+
+class Classifier(models.Model):
+    """
+    This model represents a classification model consisting of a reference to the "Middle" model and a reference to the
+
+    "Middle, Sub" model
+    """
+    created_at = models.DateTimeField(editable=False, auto_now_add=True)
+
+    name = models.CharField(max_length=255, null=False, blank=False)
+    precision = models.FloatField(null=False, blank=False, default=0)
+    recall = models.FloatField(null=False, blank=False, default=0)
+    accuracy = models.FloatField(null=False, blank=False, default=0)
```

Comment:
Kan het zo zijn dat je wel al een Classifier hebt, maar dat deze nog niet getraind is (het trainen loopt op dat moment nog asynchroon op de achtergrond)? In dat geval denk ik dat het goed is om het mogelijk te maken dat precision, recall, accuracy en de models ook leeg te laten.